data/campaigns/Under_the_Burning_Suns/scenarios/05_A_Subterranean_Struggle.cfg

```diff
@@ -583,7 +583,7 @@
     # Event 1: Spider and Ants
```

Comment:
Adding `[cancel_action]` would make sense in this event, otherwise the unit continues running into danger after spotting the enemies.